tests/test_inside_docker.sh

```diff
@@ -96,10 +54,8 @@ cp /etc/condor/config.d/99-local.conf /etc/condor-ce/config.d/99-local.conf
 # Reduce the trace timeouts
 export _condor_CONDOR_CE_TRACE_ATTEMPTS=60
 
-if [[ $BUILD_ENV == osg* ]]; then
+if [ $BUILD_ENV == osg* ] || [ $BUILD_ENV == uw_build ]; then
```

Comment:
@RyanEweSeng we only care about the `osg` and `uw_build` build environments so we can get rid of the if block altogether

Reply:
Also, for future reference, if you're writing a bash script (not /bin/sh), then you should avoid using single-bracket expressions (`[ ... ]`) and always use double-bracket because it's safer and has extra features like being able to put the or statement inside the expression. So instead of things like
```
if [ $BUILD_ENV == osg* ] || [ $BUILD_ENV == uw_build ]
```
do
```
if [[ $BUILD_ENV == osg* || $BUILD_ENV == uw_build ]]
```

Again, this is just for future reference; Brian's right that in this case we can get rid of the if block entirely.